packages/@expo/cli/src/utils/analytics/__tests__/rudderstackClient-test.ts

```diff
@@ -1,9 +1,37 @@
 import os from 'os';
 
-import { getContext } from '../rudderstackClient';
+import {
+  getContext,
+  getRudderAnalyticsClient,
+  logEvent,
+  setUserDataAsync,
+} from '../rudderstackClient';
 
 jest.mock('ci-info', () => ({ isCI: true, isPR: true, name: 'GitHub Actions' }));
 
+jest.mock('@expo/rudder-sdk-node');
+
+describe(logEvent, () => {
+  // order of these tests matters since the module retains state
```

Comment:
we should properly reset the state between tests. i think it's fine to leave this for a follow-up

Reply:
Yeah, I think it should be isolated too. Not sure what order jest runs tests in 🤷